.github/ISSUE_TEMPLATE.md

```diff
@@ -1,36 +1,46 @@
-> Please fill out this template when filing an issue.
-> All lines marked as quotes (`>`) instruct you with
-> what info we expect.
-> Remove this opening paragraph.
+> We LOVE to help with any issues or bug you have!
+
+> **Questions**: If you have questions about how to use Realm, please ask on [SO](http://stackoverflow.com/questions/ask?tags=realm) - we monitor the Realm tag.
+
+> **Feature Request**: Just fill in the first two sections below.
+
+> **Bugs**: To help you as fast as possible with an issue or bug please describe your issue and the steps you have taken to reproduce it in as many details as possible.
+>
+> Thanks for helping us help you :-)
+>
+> Please remove this and above before submitting.
```

Comment:
Not sure about this. This is quite a lot of intro text. I think it's preferable to keep it short. This duplicates a lot of the info of CONTRIBUTING.md, but this is sadly rarely read. In advance we should keep breaking the text at a fixed width, so that it is easier to read in markdown syntax, which is displayed by default.